eg/google-drive-rename

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/perl
+use strict;
+
+use feature ':5.28';
```

Comment:
This is going to break on Perl versions < 5.28

I really like using modern features but we should consider preserving backward compatibility with Perl 5.10 and later

At first glance I'm not sure to notice any 5.28 feature in the code after
please remove the line and confirm this is working as expected

Reply:
Excuse me. This was left over from `say` statements I used when debugging. It was necessary and I have removed it in my commit 542b534901b0e5aab8c9fddbc6801e04de195899.